app/controllers/concerns/pagination.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Pagination
+  extend ActiveSupport::Concern
+
+  include Pagy::Backend
+
+  included do
+    def paginated_resources_for(resources, **options)
+      pagy resources, pagination_params.merge(options)
+    end
+
+    def pagination_params
+      {
+        page: params.dig(:page, :number) || Pagy::DEFAULT[:page],
+        items: params.dig(:page, :size) || Pagy::DEFAULT[:items]
```

Comment:
Do you know if we need to specify the default values? Should it be used when we pass params as nil?
```suggestion
        page: params.dig(:page, :number),
        items: params.dig(:page, :size)
```

Reply:
@hoangmirs Thanks for spotting out. It requires when I extract to form objects because it throws the exception pagy undefined.